sev/src/lib.rs

```diff
@@ -9,22 +9,10 @@
 #![allow(missing_docs)]
 
 use sev_types::platform;
+use sev_types::platform::Build;
```

Comment:
I'm fine with this either way but I think my only nitpick is that I'd prefer to remove line 12 here and just refer to the build as `platform::Build` as I think the namespacing provided by the module adds some more precision to the readability.